### PR TITLE
Implemented `AstoAuthTokens.find()`

### PR DESCRIPTION
**Pull request by olenagerasimova**
Part of #57 
Implemented tokens search by username.

### CODE REVIEW

File: src/test/java/com/artipie/conda/asto/AstoAuthTokensTest.java [hunk 1]
```diff
@@ -54,13 +68,37 @@ void returnsTokenWhenFound() {
...
+    @Test
+    void returnsTokenByUsernameWhenFound() {
+        new TestResource(AstoAuthTokensTest.TOKENS_JSON).saveTo(this.asto, AstoAuthTokens.TKNS);
+        final String name = "alice";
+        MatcherAssert.assertThat(
+            new AstoAuthTokens(this.asto, "1 year").find(name).toCompletableFuture()
+                .join().get(),
+            new IsEqual<>(
+                // @checkstyle MagicNumberCheck (1 line)
+                new AuthTokens.TokenItem("abc123", name, Instant.ofEpochMilli(4_108_568_400_000L))
```
Comment (genryxy) [inline src/test/java/com/artipie/conda/asto/AstoAuthTokensTest.java:80]: I'd suggest to add constructor which would accept long max value implicitly when expiration date doesn't matter

**Review by genryxy — CHANGES_REQUESTED** [review]
@olenagerasimova thanks, one suggestion

File: src/test/java/com/artipie/conda/asto/AstoAuthTokensTest.java (hunk 1 above)
Comment (olenagerasimova) [reply src/test/java/com/artipie/conda/asto/AstoAuthTokensTest.java:80]: @genryxy hm, the date matters here, it's the same as in test resource file:
```
"abc123": {
    "name": "alice",
    "expire": 4108568400000
}
```

**Review by genryxy — APPROVED** [review]
@olenagerasimova thanks

**Pull request merged**